runtime/js/unix.js

```diff
@@ -294,19 +295,16 @@ function caml_unix_rmdir(name) {
 }
 
 //Provides: caml_unix_symlink
-//Requires: resolve_fs_device, caml_failwith
+//Requires: resolve_fs_device, caml_failwith, caml_jsstring_of_string
 //Alias: unix_symlink
 function caml_unix_symlink(to_dir, src, dst) {
-  var src_root = resolve_fs_device(src);
   var dst_root = resolve_fs_device(dst);
-  if (src_root.device !== dst_root.device)
-    caml_failwith("caml_unix_symlink: cannot symlink between two filesystems");
```

Comment:
I'm unsure about his one change, because the current implem will not be able to follow links across device.

Reply:
This test is wrong anyway, since `src` should be interpreted relatively to `dst`, not relatively to the current working directory.